### PR TITLE
Document JVM options in readme.md, fixes #98

### LINKED ISSUE #98: Document default JVM settings and overrides in readme

### PR DESCRIPTION
**Pull request by uschi2000**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -70,6 +70,20 @@ The `distribution` block offers the following options:
  * (optional) `javaHome` a fixed override for the `JAVA_HOME` environment variable that will
    be applied when `init.sh` is run.
 
+#### JVM Options
+The list of JVM options passed to the Java process is obtained by concatenating the following list of hard-coded
+*required options* and the list of options specified in `distribution.defaultJvmOpts`:
```
Comment (markelliot) [inline readme.md:75]: I think there's a mismatched asterisk here

File: readme.md [hunk 2]
```diff
@@ -70,6 +70,20 @@ The `distribution` block offers the following options:
...
+
+    # Hard-coded required JVM options
+    '-Djava.security.egd=file:/dev/./urandom'
```
Comment (markelliot) [inline readme.md:78]: We should probably write what these option do and why

File: readme.md (hunk 1 above)
Comment (uschi2000) [inline readme.md:75]: don't see it

File: readme.md (hunk 2 above)
Comment (uschi2000) [inline readme.md:78]: doing.

File: readme.md (hunk 1 above)
Comment (markelliot) [inline readme.md:75]: weird, think it might be a rendering bug on my phone; looks fine on my laptop

**Pull request merged**